engine/src/lib.rs

```diff
@@ -225,7 +225,7 @@ mod contract {
             .sdk_expect("ERR_INVALID_ECDSA_SIGNATURE");
 
         #[cfg(feature = "log")]
-        sdk::log(crate::prelude::format!("signer_public_key 0x{}", sender).as_str());
+        sdk::log(crate::prelude::format!("signer_address {:?}", sender).as_str());
```

Comment:
@birchmd @joshuajbouw This is now not in a consistent format with the new entries, which include a colon. Please harmonize.

Reply:
Display, as it turned out, didn’t show the whole address. Needs debug when we looked into why the address was being concatenated. Includes the 0x already too.

Reply:
Removed colons from the other messages.